### ISSUE TITLE
As of 0.2.9, cannot call single .js file

### ISSUE THREAD

**Issue by kevinharvey** [arunoda/laika#30]
This works in 0.2.8, but not 0.2.9:

```
$ TEST=1 laika tests/revisions.js

  injecting laika...
  loading phantomjs...
  loading initial app pool...
  cleaning up injected code


fs.js:654
  return binding.readdir(pathModule._makeLong(path));
                 ^
Error: ENOTDIR, not a directory 'tests/revisions.js'
    at Object.fs.readdirSync (fs.js:654:18)
    at scanFiles (/Users/xx/xx/xx/xx/node-0.10.12/lib/node_modules/laika/bin/_laika:112:22)
    at AppPool.onAppPoolReady (/Users/xx/xx/xx/xx/node-0.10.12/lib/node_modules/laika/bin/_laika:101:7)
    at AppPool.EventEmitter.emit (events.js:92:17)
    at App.createInitialPool (/Users/xx/xx/xx/xx/node-0.10.12/lib/node_modules/laika/lib/app_pool.js:30:12)
    at App.EventEmitter.emit (events.js:95:17)
    at Socket.<anonymous> (/Users/xx/xx/xx/xx/node-0.10.12/lib/node_modules/laika/lib/app.js:50:14)
    at Socket.EventEmitter.emit (events.js:117:20)
    at Socket.<anonymous> (_stream_readable.js:736:14)
    at Socket.EventEmitter.emit (events.js:92:17)
$ 
```

**Comment by arunoda** [comment]
Yes. This is a bug introduced in 0.2.9.
Let me fix this.

**Comment by arunoda** [comment]
fixed and published to `0.2.10`
